src/main/kotlin/io/perfometer/dsl/HttpDsl.kt

```diff
@@ -2,77 +2,92 @@ package io.perfometer.dsl
 
 import io.perfometer.http.*
 import java.time.Duration
+import java.util.*
 
 typealias HttpHeader = Pair<String, String>
 typealias HttpParam = Pair<String, String>
 
-class RequestBuilder(val host: String, val port: Int, val method: HttpMethod) {
+class RequestBuilder(val protocol : String,
+                     val host : String,
+                     val port : Int,
+                     val method : HttpMethod,
+                     val authorization : HttpHeader?) {
+    private var path : () -> String = { "" }
+    private val headers : MutableList<() -> HttpHeader> = mutableListOf()
+    private val params : MutableList<() -> HttpParam> = mutableListOf()
 
     val response = HttpResponse()
-    private var path: () -> String = { "" }
-    var body: () -> ByteArray = { ByteArray(0) }
-    private val headers: MutableList<() -> HttpHeader> = mutableListOf()
-    private val params: MutableList<() -> HttpParam> = mutableListOf()
+    var body : () -> ByteArray = { ByteArray(0) }
```

Comment:
According to Kotlin Code Conventions (https://kotlinlang.org/docs/reference/coding-conventions.html#colon)
space should not be placed before colon in type declarations for fields and methods.